backend/marina_app/migrations/0004_auto_20210304_0949.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.1.7 on 2021-03-04 14:49
+
```

Comment:
Is there a way we can remove these auto generated files?

Reply:
I definitely think there is a way, honestly I'm still not sure the safe way to do that right now